relay-server/src/services/store.rs

```diff
@@ -1390,6 +1390,8 @@ struct SpanKafkaMessage<'a> {
     #[serde(default)]
     is_segment: bool,
 
+    #[serde(default, skip_serializing_if = "none_or_empty_object")]
+    data: Option<&'a RawValue>,
```

Comment:
I was confused where this field is set, then saw that it's because of the useless serialization roundtrip 🤣

Reply:
Yeah we're basically trying to make the roundtrip as cheap as possible.